### PR TITLE
[FEATURE] DataAssistants Example Notebook - Spark

### PR DESCRIPTION
**Pull request by Shinnnyshinshin**
Changes proposed in this pull request:
- Update `tests/test_fixtures/rule_based_profiler/example_notebooks/DataAssistants_Instantiation_And_Running-OnboardingAssistant-Pandas.ipynb` to include the most recent workflow (which was first shown in the sql `DataAssistants` example notebook)
### The Workflow looks like this 
- Running DataAssistant on 2019 Taxi Data (our "training set")
- Running Checkpoint on 2020 January Taxi Data (our "test set")
- Update Appendix to include parameter descriptions for `DataAssistant`, and how they can be adjusted / modified
- **Note**: Since `Pandas` does not currently support being able to define a schema, there is a note that describes this limitation. Schemas are supported by the sql example (in the database), and `Spark` (soon) 

After submitting your PR, CI checks will run and @cla-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in github issues or slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-

### Definition of Done
Please delete options that are not relevant.

- [ ] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style)
- [ ] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added [unit tests](https://docs.greatexpectations.io/docs/contributing/contributing_test#writing-unit-and-integration-tests) where applicable and made sure that new and existing tests are passing.
- [ ] I have run any local integration tests and made sure that nothing is broken.

Thank you for submitting!

### CODE REVIEW

**Comment by Shinnnyshinshin** [comment]
trying this one again too 😓
